articles/overview/what-is-quantum-computing.md

```diff
@@ -0,0 +1,54 @@
+---
+title: What is quantum computing?
+description: Learn what quantum computing is, and what a quantum computer can do
+author: natke
+ms.author: nakersha
+ms.date: 10/14/2019
+ms.topic: article
+uid: microsoft.quantum.overview.what
+---
+
+# What is quantum computing?
+
+Quantum computing is the application of quantum physics to the field of computer science.
+
+The theory of quantum physics posits that matter, at a quantum level is simultaneously in multiple states. And those many states interfere with each other like waves in a tide pool.
```

Comment:
```suggestion
The theory of quantum physics posits that matter, at a quantum level can be in a superposition of multiple classical states. And those many states interfere with each other like waves in a tide pool.
```
We have to be careful with the term "state". Saying that matter can be in "multiple states" can lead to misconceptions because we physicists usually refer to "state of the system" as the quantum superposition of multiple classical states. Therefore, the system is only in one state (that is a specific superposition of several classical states).